### DISCUSSION TITLE
How to load GeoTIFF file without Painting color scheme to bitmap?

### DISCUSSION THREAD

**Discussion by abt5555** [DotSpatial/DotSpatial#1531] [Q&A]
Hello

I would like to load a geotiff file to the map. I use the below code:

`map1.AddLayer(openDlg.FileName);`

It loads but because of 'Painting color scheme to bitmap' process it loads very slow and changes the colors of some geotiff files. 

Is there any way to cancel this process. I use gdal plugin.

Thank you

**Comment by jany-tenaj** [comment]
Can you attach the file you're loading and an image of what it should look like and what it does look like?

**Comment by abt5555** [comment]
Hello,

I attached the files. Please let me know if you have problem to see them. Thank you.

dsdsd.zip is the file I am loading.

gm.png is the what it should look like.

ass.png is the what it does look like.

[dsdsd.zip](https://github.com/DotSpatial/DotSpatial/files/13039447/dsdsd.zip)
![gm](https://github.com/DotSpatial/DotSpatial/assets/14254710/ae77a9fc-f7de-43e2-959f-660776b8647c)
![ass](https://github.com/DotSpatial/DotSpatial/assets/14254710/8abd9d41-ff78-4f2e-9e89-6ab8d9881340)

**Reply by jany-tenaj** [reply]
Your problem comes from the fact that your file has more than one band. That is currently not supported by the gdal plugin. Because only one band is read you get the wrong colors. 

I added some lines that skip the loading for multiband rasters so InRamImageLayer gets a chance to try and load those files and your file looks correct afterwards.

So check out the latest commit from the master branch to see whether it works correctly for you.
